controller/v1/base/health.go

```diff
@@ -7,11 +7,14 @@ import (
 	"github.com/opensaucerer/gotemplate/typing"
 )
 
-func HealthCheck(w http.ResponseWriter, r *http.Request) {
-	data := typing.Health{
-		Version:     "0.0.1",
-		Status:      true,
-		Description: "Raid2Earns. Leaders create raids and pawns pick 'em up.",
+//HealthCheck checks health information of the app in relation to a specific version
```

Comment:
`Health` not `HealthCheck`

Reply:
I think you checked the wrong version. I already changed it since lastweek